File: mirai-core/src/main/java/net/mamoe/mirai/event/events/group/GroupEvent.java
package net.mamoe.mirai.event.events.group;

import net.mamoe.mirai.Bot;
import net.mamoe.mirai.contact.Group;
import net.mamoe.mirai.event.events.bot.BotEvent;
import org.jetbrains.annotations.NotNull;

/**
 * @author Him188moe
 */
public abstract class GroupEvent extends BotEvent {
    private final Group group;

    public GroupEvent(Bot bot, Group group) {
        super(bot);
        this.group = group;
    }

    @NotNull
    public Group getGroup() {
        return group;
    }
}

File: mirai-core/src/main/java/net/mamoe/mirai/event/events/group/GroupEvent.kt
package net.mamoe.mirai.event.events.group

import net.mamoe.mirai.Bot
import net.mamoe.mirai.contact.Group
import net.mamoe.mirai.event.events.bot.BotEvent

/**
 * @author Him188moe
 */
abstract class GroupEvent(bot: Bot, val group: Group) : BotEvent(bot)